code/modules/research/designs/stock_parts_designs.dm

```diff
@@ -113,10 +113,10 @@
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_SCIENCE
 
 //Maipulators
-/datum/design/micro_mani
-	name = "Micro Manipulator"
+/datum/design/basic_manipulator
+	name = "Basic Manipulator"
 	desc = "A stock part used in the construction of various devices."
-	id = "micro_mani"
+	id = "basic_manipulator"
```

Comment:
you sure you still want this change in if you took a different solution?

Reply:
Yeah, it puts them next to the other tier 1 parts in the stock part list and names them similarly, which helps new players understand they're tier 1.